src/components/NavigationBar/index.tsx

```diff
@@ -66,7 +66,7 @@ const NavigationBar = ({
             activeClassName="usa-current"
             className="easi-nav__link"
             onClick={() => toggle(false)}
-            exact
+            exact={route.link === '/'}
```

Comment:
This works for most routes, only if the root path remains the same.  For IT Governance (/system/making-a-request) and Section 508 (/508/making-a-request), those paths are not at the root of its children.  This seems like a design flaw that we may want to revisit.  For example, drilling into 'Start a new request' on IT Governance page, routes to path not covered by the 'exact' prop, where the new route is '/system/request-type' - which is a sibling route, not child of '/system/making-a-request'.  One option is to make the main navigation routes each being with a more central path ('/system' instead of '/system/making-a-request') This change may a substantial impact on the code base and testing.  The other may be to create a custom function to pass to the 'exact' prop of the React link.  I think the first way is cleaner, but more impact, second is quicker and less risk.

Reply:
@patrickseguraoddball I like the idea of cleaning up urls and simplifying. What kind of checks would we need with the existing urls?

Reply:
I think the scope of modifying these two base urls will require some cypress/unit tests changes to ensure the intended paths are hit once updated.  I would do a full scoping to see where these changes would affect areas of the codebase.  I think it should be fairly minimal, but I expect a few tests to have to be reflected in these changes.

Reply:
@ClayBenson94 would you see any further impact on modifying the two base paths for '/system/making-a-request' and '/508/making-a-request'.  I think there may be some caution if anyone has bookmarked specific routes, etc.

Reply:
Can we simplify the paths by making them hierarchical and then make redirect the old paths?